Processing /Users/sidgurung/Desktop/ZELDA/ZELDA_II
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: setuptools>66 in /opt/anaconda3/lib/python3.12/site-packages (from Lya_zelda_II==0.0.1) (69.5.1)
Requirement already satisfied: scikit-learn==1.5 in /Users/sidgurung/.local/lib/python3.12/site-packages (from Lya_zelda_II==0.0.1) (1.5.0)
Requirement already satisfied: pyswarms==1.3.0 in /Users/sidgurung/.local/lib/python3.12/site-packages (from Lya_zelda_II==0.0.1) (1.3.0)
Requirement already satisfied: emcee==3.0.2 in /Users/sidgurung/.local/lib/python3.12/site-packages (from Lya_zelda_II==0.0.1) (3.0.2)
Requirement already satisfied: numpy<2.0 in /Users/sidgurung/.local/lib/python3.12/site-packages (from Lya_zelda_II==0.0.1) (1.26.4)
Requirement already satisfied: scipy in /opt/anaconda3/lib/python3.12/site-packages (from pyswarms==1.3.0->Lya_zelda_II==0.0.1) (1.13.1)
Requirement already satisfied: matplotlib>=1.3.1 in /opt/anaconda3/lib/python3.12/site-packages (from pyswarms==1.3.0->Lya_zelda_II==0.0.1) (3.8.4)
Requirement already satisfied: attrs in /opt/anaconda3/lib/python3.12/site-packages (from pyswarms==1.3.0->Lya_zelda_II==0.0.1) (23.1.0)
Requirement already satisfied: tqdm in /opt/anaconda3/lib/python3.12/site-packages (from pyswarms==1.3.0->Lya_zelda_II==0.0.1) (4.66.4)
Requirement already satisfied: future in /Users/sidgurung/.local/lib/python3.12/site-packages (from pyswarms==1.3.0->Lya_zelda_II==0.0.1) (1.0.0)
Requirement already satisfied: pyyaml in /opt/anaconda3/lib/python3.12/site-packages (from pyswarms==1.3.0->Lya_zelda_II==0.0.1) (6.0.1)
Requirement already satisfied: joblib>=1.2.0 in /opt/anaconda3/lib/python3.12/site-packages (from scikit-learn==1.5->Lya_zelda_II==0.0.1) (1.4.2)
Requirement already satisfied: threadpoolctl>=3.1.0 in /Users/sidgurung/.local/lib/python3.12/site-packages (from scikit-learn==1.5->Lya_zelda_II==0.0.1) (3.5.0)
Requirement already satisfied: contourpy>=1.0.1 in /Users/sidgurung/.local/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (1.3.0)
Requirement already satisfied: cycler>=0.10 in /opt/anaconda3/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (0.11.0)
Requirement already satisfied: fonttools>=4.22.0 in /opt/anaconda3/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (4.51.0)
Requirement already satisfied: kiwisolver>=1.3.1 in /opt/anaconda3/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (1.4.4)
Requirement already satisfied: packaging>=20.0 in /opt/anaconda3/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (23.2)
Requirement already satisfied: pillow>=8 in /opt/anaconda3/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (10.3.0)
Requirement already satisfied: pyparsing>=2.3.1 in /opt/anaconda3/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (3.0.9)
Requirement already satisfied: python-dateutil>=2.7 in /opt/anaconda3/lib/python3.12/site-packages (from matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (2.9.0.post0)
Requirement already satisfied: six>=1.5 in /opt/anaconda3/lib/python3.12/site-packages (from python-dateutil>=2.7->matplotlib>=1.3.1->pyswarms==1.3.0->Lya_zelda_II==0.0.1) (1.16.0)
Building wheels for collected packages: Lya_zelda_II
  Building wheel for Lya_zelda_II (setup.py): started
  Building wheel for Lya_zelda_II (setup.py): finished with status 'done'
  Created wheel for Lya_zelda_II: filename=Lya_zelda_II-0.0.1-py3-none-any.whl size=57212999 sha256=4cfeb9cce38ca18abb25496aa01ca8bebba7967ab34d38fc666db643efab7746
  Stored in directory: /private/var/folders/77/84x2925s3gbg_r5vrz5g2flw0000gn/T/pip-ephem-wheel-cache-s97c9ipn/wheels/ad/db/77/17883b229b842d49d4ec73dad034dc6eebb00a22863421cb22
Successfully built Lya_zelda_II
Installing collected packages: Lya_zelda_II
  Attempting uninstall: Lya_zelda_II
    Found existing installation: Lya_zelda_II 0.0.1
    Uninstalling Lya_zelda_II-0.0.1:
      Successfully uninstalled Lya_zelda_II-0.0.1
Successfully installed Lya_zelda_II-0.0.1
